### PR TITLE
blog: gui v0.17.3.1 release notes

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: _posts/2021-01-06-monero-GUI-0.17.3.1-released.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Fix startup crash on Apple Silicon (#[3804](https://github.com/monero-project/monero-gui/pull/3804))
+- Fix wizard password not resetting after wallet creation (#[3795](https://github.com/monero-project/monero-gui/pull/3795))
+- Update translations (#[3811](https://github.com/monero-project/monero-gui/pull/3811))
+- Minor bug fixes and UI improvements
+
+The complete list of changes is [available on GitHub](https://github.com/monero-project/monero-gui/compare/v0.17.3.0...v0.17.3.1), along with [the source code](https://github.com/monero-project/monero-gui/tree/v0.17.3.1).
+
+# Contributors for this Release
+
+This release was the direct result of 65 people who worked, largely unpaid and altruistically, to put out 93 commits containing 31902 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
```
Comment (erciccione) [inline _posts/2021-01-06-monero-GUI-0.17.3.1-released.md:31]: 65?

**Review by erciccione — COMMENTED** [review]
Could you add `GUI` somewhere in the commit? It would make more clear that this is a GUI-only release

File: _posts/2021-01-06-monero-GUI-0.17.3.1-released.md (hunk 1 above)
Comment (selsta) [reply _posts/2021-01-06-monero-GUI-0.17.3.1-released.md:31]: Forgot to update it :)

**Review by erciccione — APPROVED** [review]
Hashes to be verified.

**Comment by binaryFate** [comment]
Hashes verified

**Pull request merged**